projects/ids-enterprise-ng/src/lib/pager/soho-standalone-pager.component.ts

```diff
@@ -31,47 +31,83 @@ export class SohoStandalonePagerComponent implements AfterViewInit, AfterViewChe
 
   @Input() set showFirstButton(showFirstButton: boolean) {
     this.options.showFirstButton = showFirstButton;
-    this.updateRequired = !!this.pager;
+    if (this.pager) {
+      this.pager.showButton('first', showFirstButton);
+    } else {
+      this.updateRequired = !!this.pager;
```

Comment:
A bit confused with setting `this.updateRequired` to `false` when the pager is not built yet. Shouldn't it be `this.updateRequired = true;` since we'd want to update the pager via AfterViewCheck once it's built?

Reply:
@nbcp yes, I think that is more clear. I'll update.